enterprise/wan-replication/simulating-max-idle/README.md

```diff
@@ -0,0 +1,53 @@
+## About
+
+During wan replication, only write events are replicated. This means
+that if you use max idle feature with wan replication, entries on the
+target cluster may expire, even if the source receives the read events.
+
+As a workaround for this issue, if you don't care about:
+
+ - Entries may live longer than configured max idle or,
+ - You calculate the expiration yourself
+
+You can use an `EntryProcessor`.
+
+## Detail
+
+Ideally to solve the issue, we can do write after each read. But that
```

Comment:
```suggestion
Ideal solution is wan-replicating reads as in writes. But that
```

Reply:
I was trying to explain that a very naive approach would be to call put after every get. I tried to explain that better https://github.com/hazelcast/hazelcast-code-samples/pull/560/commits/664e5544ce8a82b46e4878d5ed9c335cc0f2d1c7